lib/topological_inventory/providers/common/operations/source.rb

```diff
@@ -116,10 +116,16 @@ def update_endpoint(status, error_message)
 
         def endpoint
           @endpoint ||= api_client.fetch_default_endpoint(source_id)
+        rescue StandardError => e
+          logger.availability_check("Failed to fetch Endpoint for Source #{source_id}, Endpoint #{endpoint}: #{e.message}", :error)
+          return
```

Comment:
@lindgrenj6 Is the `return` to prevent whatever is sent back from `logger.availability_check` from being returned to the caller?  If so, what does `logger.availability_check` return?

Reply:
Well it looks like MIQ loggers inherits from the base logger - so doing a quick test it returns true: 
```
>> Logger.new(STDOUT).info("test") #=> true
I, [2020-07-06T14:45:03.236519 #149848]  INFO -- : test
```

but I _need_ it to return `nil` that way the endpoint isn't present: https://github.com/RedHatInsights/topological_inventory-providers-common/blob/5c04f92462815485bffcb4f798bb82a490e28a82/lib/topological_inventory/providers/common/operations/source.rb#L69